core/main/src/service/apps/delegated_launcher_handler.rs

```diff
@@ -192,12 +204,20 @@ impl DelegatedLauncherHandler {
                     resp = self.set_state(&app_id, state).await;
                 }
                 AppMethod::Launch(launch_request) => {
+                    if self.platform_state.has_internal_launcher() {
+                        // When using internal launcher extension the NavigationIntent structure will get untagged we will use the original
+                        // intent in these cases to avoid loss of data
+                        self.platform_state.app_manager_state.store_intent(
+                            &launch_request.app_id,
+                            launch_request.get_intent().clone(),
+                        );
+                    }
```

Comment:
Does this mean that even when building the equivalent of a "gateway_with_launcher" type ripple (e.g. for FC-style builds), we're including and building `DelegatedLauncherHandler`?

Reply:
yes DelegatedLauncherHandler is bundled by default. Launcher extension is included or not depending on the distributor